File: chroma-league/src/test/java/net/booone/chromaleague/hud/animations/AllyCloudDragonKillAnimationTest.java
package net.booone.chromaleague.hud.animations;

import net.booone.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class AllyCloudDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playAllyCloudDragonKillAnimation() {
        new AnimationTester().testAnimation(new AllyCloudDragonKillAnimation());
    }
}

File: test/hud/animations/AllyCloudDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {AllyCloudDragonKillAnimation} from '../../../src/hud/animations/AllyCloudDragonKillAnimation.js';

/**
 * Integration test for ally cloud dragon kill animation on actual hardware.
 */
describe('AllyCloudDragonKillAnimation Integration', () => {
    test('playAllyCloudDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new AllyCloudDragonKillAnimation());
    }, 30000);
});
